content/ri/release-notes/v.2.42.0.md

```diff
@@ -0,0 +1,25 @@
+---
+Title: RedisInsight v2.42.0, January 2024
+linkTitle: v2.42.0 (January 2024)
+date: 2024-01-30 00:00:00 +0000
+description: RedisInsight v2.42
+weight: 1
+aliases: /ri/release-notes/v2.42.0/
+         /ri/release-notes/v2.42.0.md
+---
+## 2.42 (January 2024)
+This is the General Availability (GA) release of RedisInsight 2.42.
+
+### Highlights
+- Introducing a new dedicated developer enablement area! Explore Redis capabilities and learn easily, among other things, how to leverage the native JSON data structure supporting structured querying and full-text search, including vector similarity search for your AI use cases. Browse the tutorials offline or leverage the in-app provisioning of a free Redis Cloud database to try them interactively.
+- RedisInsight is now available on Docker. Check out our [Docker repository](https://hub.docker.com/repository/docker/redis/redisinsight/general) if that’s your preferred platform. 
+
+
+### Details
+
+**Features and improvements**
+- [#2724](https://github.com/RedisInsight/RedisInsight/pull/2724), [#2752](https://github.com/RedisInsight/RedisInsight/pull/2752), [#2965](https://github.com/RedisInsight/RedisInsight/pull/2965) Introducing a dedicated developer enablement area. Dive into interactive tutorials and level up your Redis game even without a database connected. Start exploring tutorials by clicking on the "Insights" button located in the top-right corner. Interactive tutorials may impact your database by altering data, thus avoid running them in a production environment. For an optimal tutorial experience, create a free [Redis Cloud](https://redis.com/try-free/?utm_source=redisinsight&utm_medium=main&utm_campaign=redisinsight_release_notes) database.
```

Comment:
```suggestion
- [#2724](https://github.com/RedisInsight/RedisInsight/pull/2724), [#2752](https://github.com/RedisInsight/RedisInsight/pull/2752), [#2965](https://github.com/RedisInsight/RedisInsight/pull/2965) Introducing a dedicated developer enablement area. Dive into interactive tutorials and level up your Redis game even without a database connected. Start exploring tutorials by clicking on the "Insights" button located in the top-right corner. Because interactive tutorials can alter data in your database, avoid running them in a production environment. For an optimal tutorial experience, create a free [Redis Cloud](https://redis.com/try-free/?utm_source=redisinsight&utm_medium=main&utm_campaign=redisinsight_release_notes) database.
```